nss-day-cohort-44/nutshell-sun-satyrs
# ts-chat-requirement-3

# Description
Fixes use case:
- [x] Given a user wants to delete a previous message of theirs 
When the user performs a gesture on the delete affordance in the chat message 
Then that specific message should be removed from the database and the UI

My changes:
- Created a GlobalMessage function in `Message.js` that is responsible for rendering every user's messages, except for the `activeUser`'s, without a Delete button. This allows the `activeUser` to only delete their own messages.
- Added globalMessages variable in `MessageList.js` that stores messages from every user except for the `activeUser`.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
1. `git fetch --all`
1. `git checkout ts-chat-requirement-3`
1. `serve`
1. Verify that you can now see every user's messages, but you can only delete messages of the `activeUser`.

# Checklist:
